src/css/globals/utility.css

```diff
@@ -15,3 +15,36 @@ vars for reference:
 .no-pointer {
   pointer-events: none;
 }
+
```

Comment:
Add these additions are required to replace inline style attributes in marketing's header and footer with class attributes.